tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -216,6 +217,16 @@ def raise_ie_not_found():
         log.error(e)
         raise_ie_not_found()
 
+    if 'compress_to_fp16' in argv and argv.compress_to_fp16 and \
+            'data_type' in argv and argv.data_type in ['FP32', 'float']:
+        raise Error(f"Both --compress_to_fp16 and --data_type={argv.data_type} are defined. "
+                    f"Specify either compress_to_fp16 or data_type={argv.data_type}")
+
+    if ('compress_to_fp16' not in argv or argv.compress_to_fp16 is None) \
+            and ('data_type' not in argv or argv.data_type is None):
+        argv.compress_to_fp16 = True
+        print(get_compression_message())
```

Comment:
Let us have it here: https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/openvino/tools/mo/main.py#L50

Reply:
done